harness/determined/common/util.py

```diff
@@ -5,7 +5,8 @@
 import platform
 import random
 import sys
-from typing import IO, Any, Callable, Iterator, Sequence, TypeVar, Union, overload
+from copy import deepcopy
```

Comment:
can you import the module instead of the function?

This is the standard we (try to) use in new code, and has been for a couple years.  There's still lots of old that hasn't been modified during that time that doesn't follow this rule, but it's still much better if we follow it.

Reply:
Sure, can do. I was curious about the motivation behind this? Is it just so that function calls in the code specify the full  module.submodule.function()?